dom0/sd-svs.sls

```diff
@@ -45,6 +45,14 @@ sd-svs:
     - require:
       - qvm: sd-svs-buster-template
 
+# The private volume should be 10GB in size
+sd-svs-private-volume-size:
+  cmd.run:
+    - name: >
+        qvm-volume resize sd-svs:private 10GiB
```

Comment:
:heavy_check_mark: It seems like qvm.volume is not implemented in salt, so we must use cmd.run here